code_of_conduct.md

```diff
@@ -47,6 +48,7 @@ The following behaviors are considered harassment and are unacceptable within ou
 *   Deliberate intimidation, stalking or following (online or in person).
 *   Advocating for, or encouraging, any of the above behavior.
 *   Sustained disruption of community events, including talks and presentations.
+*   Collecting or sharing personal information (including names, emails, avatars, handles, nicknames, messages, emails, or other biographical or identifying information) without explicit and prior consent.
```

Comment:
ohh.. "emails" is twice

Reply:
also, maybe it should be "email addresses"?

Reply:
I’m out but will fix this @nathany. Good catch!